lib/fluent/plugin/storage_local.rb

```diff
@@ -64,7 +64,8 @@ def configure(conf)
             end
           end
         elsif root_dir = owner.plugin_root_dir
-          @path = File.join(root_dir, 'storage.json')
+          basename = (conf.arg && !conf.arg.empty?) ? "storage.#{conf.arg}.json" : "storage.json"
```

Comment:
This `conf.arg` will be given by :usage in storage_create?
Or users should specify `<storage args>` by themselves?